File: C Modules/Assignments/Assignment no 7/Question-1/Type_3/Assignment no 2/Question-2.c
#include<stdio.h>
int traiangle()
{
    int a,b,c;
    int *ptr1=&a;
    int *ptr2=&b;
    int *ptr3=&c;

    printf("enter the 3 sides of triangle:");
    scanf("%d %d %d", ptr1, ptr2, ptr3);
      if (*ptr1+*ptr2<=*ptr3||*ptr2+*ptr3<=*ptr1||*ptr1+*ptr3<=*ptr2)
    {
        return 0;
    }
    else if (*ptr1 == *ptr2 && *ptr2 == *ptr3) {
         return 1;
    }
    else if (*ptr1 == *ptr2 || *ptr2 == *ptr3 || *ptr1 == *ptr3) {
        return 2;
    }
    else {
        return 3;
    }
}
int main()
{
   int result=traiangle();
    if (result==0)
    {
        printf("Not a valid traiangle:\n");
        /* code */
    }
    else if (result==1) {
        printf("Equilateral triangle\n");
    }
    else if (result==2) {
        printf("Isosceles triangle\n");
    }
    else {
        printf("Scalene triangle\n");
    }
   return 0; 
}

File: Java Module/Assignments in c to java/Assignment-02/Question_2.java
public class Question_2 {
public static void main(String[] args) {
    
int a=3,b=2,c=3;
    // printf("enter the 3 sides of triangle:");
    // scanf("%d %d %d", &a, &b, &c);
    if (a+b<=c||b+c<=a||a+c<=b)
    {
        System.out.printf("Not a valid traiangle:\n");
        /* code */
    }
    else if (a == b && b == c) {
        System.out.printf("Equilateral triangle\n");
    }
    else if (a == b || b == c || a == c) {
        System.out.printf("Isosceles triangle\n");
    }
    else {
        System.out.printf("Scalene triangle\n");
    }
    
}
}



